### PR TITLE
Moved some dependencies into test scope only in helidon-webserver-tyrus

### PR DESCRIPTION
**Pull request by spericas**
Issue #3411.

Signed-off-by: Santiago Pericasgeertsen <santiago.pericasgeertsen@oracle.com>

### CODE REVIEW

**Comment by spericas** [comment]
```
[INFO] +- org.glassfish.tyrus:tyrus-client:jar:1.17:test
[INFO] +- org.glassfish.tyrus:tyrus-container-grizzly-client:jar:1.17:test
[INFO] |  +- org.glassfish.grizzly:grizzly-framework:jar:2.3.22:test
[INFO] |  \- org.glassfish.grizzly:grizzly-http-server:jar:2.3.22:test
[INFO] |     \- org.glassfish.grizzly:grizzly-http:jar:2.3.22:test
```

File: dependencies/pom.xml [hunk 1]
```diff
@@ -510,10 +510,25 @@
                 <artifactId>tyrus-client</artifactId>
                 <version>${version.lib.tyrus}</version>
             </dependency>
+            <dependency>
+                <groupId>org.glassfish.tyrus</groupId>
+                <artifactId>tyrus-container-grizzly-client</artifactId>
```
Comment (romain-grecourt) [inline dependencies/pom.xml:515]: Is this dependency meant to be used by end-users ? If not they can be moved to the top level pom.

Comment (spericas) [reply dependencies/pom.xml:515]: They would use it only for testing WS, just like we do now. WDYT?

Comment (romain-grecourt) [reply dependencies/pom.xml:516]: In that case, that is the right place for this.

**Review by romain-grecourt — APPROVED** [review]

**Pull request merged**